src/utils/components/ListPageFilter/ListPageFilter.tsx

```diff
@@ -117,6 +150,10 @@ const ListPageFilter: FC<ListPageFilterProps> = ({
 
   const selectedSearchFilter = searchFilters?.find((f) => f.type === searchType);
 
+  const showSearchFilters = Object.keys(filterDropdownItems).length !== 0;
+
+  const showSearchFiltersDropdown = Object.keys(filterDropdownItems).length > 1;
```

Comment:
feels strange.. 1 check if its 1 or more the other check more then one? can u make it one? or use showSearchFiltersDropdown = showSearchFilter > 1